aequilibrae/paths/results/assignment_results.py

```diff
@@ -51,6 +51,11 @@ def __init__(self):
         self.lids = None
         self.direcs = None
 
+        # save path files. Need extra metadata for file paths
+        self.save_path_file = False
+        self.path_file_dir = None
+        self.write_feather = True  # we use feather as default, parquet is slower but with better compression
```

Comment:
Is the distinction between parquet and Feather something we need to know when we will read the data?  If so, we need to include this in the Assignment report (the one where we write the betas and convergence report to)

Reply:
good point, I'll add it. At this point both formats are implemented so I'll just leave them, but I think we could also just remove parquet and only support feather.